### PR TITLE
Start ChangeLog

### PR DESCRIPTION
**Pull request by dns2utf8**
I would like to start a ChangeLog. What do you think of this?

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+[Unreleased]: https://github.com/coredump-ch/coredumpbot/compare/v0.2.6...HEAD
+[0.2.6]: https://github.com/coredump-ch/coredumpbot/compare/0.2.5...0.2.6
+[0.2.5]: https://github.com/coredump-ch/coredumpbot/compare/0.2.4...0.2.5
+[0.2.4]: https://github.com/coredump-ch/coredumpbot/compare/0.2.3...0.2.4
+[0.2.3]: https://github.com/coredump-ch/coredumpbot/compare/0.2.2...0.2.3
+[0.2.2]: https://github.com/coredump-ch/coredumpbot/compare/0.2.1...0.2.2
+[0.2.1]: https://github.com/coredump-ch/coredumpbot/compare/0.2.0...0.2.1
+[0.2.0]: https://github.com/coredump-ch/coredumpbot/compare/0.1.0...0.2.0
+[0.1.0]: https://github.com/coredump-ch/coredumpbot/compare/f6b6d059d86...0.1.0
```
Comment (dbrgn) [inline CHANGELOG.md:66]: Ah, using references. That's great, because it doesn't impair the readability of the main changelog!

Comment (dbrgn) [inline CHANGELOG.md:66]: Woah, and you link to the diff. I didn't even know that was possible. Awesome!

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
+# Changelog
```
Comment (dbrgn) [inline CHANGELOG.md:1]: I'd add a note about semantic versioning. And adding categories to the messages _might_ be useful :)

**Comment by dbrgn** [comment]
@dns2utf8 if you do an adjustment in a pull request, please leave a comment in the future. otherwise we'll get no notification about the change and will forget about the pull request :)

go ahead and merge.

File: CHANGELOG.md (hunk 1 above)
Comment (schmijos) [inline CHANGELOG.md:66]: I also really like that. If you event want to go further you could write tests for it ;-)
https://github.com/bbatsov/rubocop/blob/master/spec/project_spec.rb#L27

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+## [0.2.0] - 2015-11-12
+
+ - Add /crowd to answer people_now_present
+
+## [0.1.0] - 2015-11-11
+
+ - Initial Release
+
+[Unreleased]: https://github.com/coredump-ch/coredumpbot/compare/v0.2.6...HEAD
```
Comment (schmijos) [inline CHANGELOG.md:58]: What about linking to the _master_ reference? Isn't the `HEAD` pointer different for each branch?

Comment (dbrgn) [inline CHANGELOG.md:58]: I assume Github uses the HEAD of the default branch.

Btw, the `v` in there is wrong, the tags don't use a leading `v`.

Comment (schmijos) [inline CHANGELOG.md:58]: **Offtopic?:**

Tags use what you define. But I'm still not so sure about what actually should be used. There where quite some use cases where it was really useful for me that tags were using a `v` prefix.
- here it seems clear: http://semver.org/spec/v1.0.0.html#tagging-specification-semvertag
- but: https://github.com/mojombo/semver/issues/1
- and: https://github.com/mojombo/semver/issues/204

So version 2 of semver doesn't specify it anymore.

Comment (dbrgn) [inline CHANGELOG.md:58]: I personally like the `v` prefix, but @dns2utf8 seems to have decided against them (see upper part of CHANGELOG). Just keep it consistent :)

Comment (dns2utf8) [inline CHANGELOG.md:58]: I have been trying to remove the v0.1.0 Tag, but I had no luck.

**Pull request merged**